test/integration/puppetlabs/services/certificate_authority/certificate_authority_int_test.clj

```diff
@@ -482,6 +482,70 @@
         (spit infra-crl-path infra-crl-content)
         (spit infra-serials-path infra-serials-content))))
 
+(deftest ^:integration clean-infrastructure-certs
+  (let [master-conf-dir (str test-resources-dir "/infracrl_test/master/conf")
+        subject1 "compile-master"
+        ;; We're going to pretend this is an infra cert for this test
+        subject2 "agent-node"
+        hostcrl-path (str master-conf-dir "/ssl/crl.pem")
+        hostcrl-content (slurp hostcrl-path)
+        full-crl-path (str master-conf-dir "/ssl/ca/ca_crl.pem")
+        full-crl-content (slurp full-crl-path)
+        infra-crl-path (str master-conf-dir "/ssl/ca/infra_crl.pem")
+        infra-crl-content (slurp infra-crl-path)
+        infra-serials-path (str master-conf-dir "/ssl/ca/infra_serials")
+        infra-serials-content (slurp infra-serials-path)
+        infra-inventory-path (str master-conf-dir "/ssl/ca/infra_inventory.txt")
+        infra-inventory-content (slurp infra-inventory-path)
+        cert1-path (ca/path-to-cert (str master-conf-dir "/ssl/ca/signed") subject1)
+        cert1 (utils/pem->cert cert1-path)
+        cert2-path (ca/path-to-cert (str master-conf-dir "/ssl/ca/signed") subject2)
+        cert2 (utils/pem->cert cert2-path)]
+    ;; Add another cert to the infra inventory
+    (spit infra-inventory-path (str infra-inventory-content "agent-node"))
+    (bootstrap/with-puppetserver-running-with-mock-jrubies
+     "JRuby mocking is safe here because all of the requests are to the CA
+     endpots, which are implemented in Clojure."
```

Comment:
lol
```suggestion
     endpoints, which are implemented in Clojure."
```

Reply:
Turns out this was all over the place due to copy-pasta.